### PR TITLE
Fix/update xil telematics cfg

### PR DESCRIPTION
**Pull request by adev4a**
# PR Details
This PR updates the xil-cloud-telematics config with changes from xil-cloud as well as fixes issues with the telematics module deployment. A volume is added to the vehicle calibration directory and the environment variable name is fixed.

Note: The only difference between the xil-cloud image and xil-cloud-telematics image is the added carma-vehicle-nats-bridge_1 service that will enable users to optionally stream telematics data for visualization. The xil-cloud-telematics image should be kept updated with the xil-cloud changes in order to work well with CDASim included services.

## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: xil_cloud_telematics/carma.config.js [hunk 1]
```diff
@@ -41,34 +41,16 @@ CarmaJS.registerNamespace("CarmaJS.Config");
 
 CarmaJS.Config = (function () {
         //Private variables
-        var ip = '127.0.0.1'; //'192.168.88.10'; 192.168.32.146;
-        var refresh_interval = 30; //30 seconds
-        var ros_connect_wait = 5000; //5 miliseconds to wait for platform to launch and ros to connect.
-        var ros_connect_retry = 24; //# of times to wait. Total 2 minutes
-        var speed_limit = 30; //Default Speed Limit
```
Comment (kjrush) [inline xil_cloud_telematics/carma.config.js:48]: Is there a reason for removing these? They're present in the physical vehicle configs, and while I'm not totally sure what they do, I don't think we intentionally removed these in sim?

File: xil_cloud_telematics/carma.config.js [hunk 2]
```diff
@@ -41,34 +41,16 @@ CarmaJS.registerNamespace("CarmaJS.Config");
...
+        var ip = '127.0.0.1';
+
         //Private methods
         //Creating functions to prevent access by reference to private variables
         var getIP = function() {
             return ip;
         };
-        var getRefreshInterval = function(){
```
Comment (kjrush) [inline xil_cloud_telematics/carma.config.js:54]: Same with these functions.

File: xil_cloud_telematics/carma_docker.launch [hunk 3]
```diff
@@ -56,9 +56,6 @@
   <!-- Launch the CARMA-CARLA simulation -->
   <arg name="simulation_mode" default="true"/>
 
-  <!-- Rosbag Flag -->
```
Comment (kjrush) [inline xil_cloud_telematics/carma_docker.launch:59]: This is replaced by the `use_rosbag` parameters above right? Is this just part of the ros2 changes  we missed in the XIL config?

File: xil_cloud_telematics/docker-compose-background.yml [hunk 4]
```diff
@@ -1,29 +0,0 @@
-#  Copyright (C) 2018-2021 LEIDOS.
```
Comment (kjrush) [inline xil_cloud_telematics/docker-compose-background.yml:1]: This file doesn't seem to have existed in the xil_cloud config, was the xil_cloud_telematics config started from a different template?

**Review by kjrush — APPROVED** [review]

File: xil_cloud_telematics/carma_docker.launch (hunk 3 above)
Comment (adev4a) [reply xil_cloud_telematics/carma_docker.launch:59]: Yes, this is correct

File: xil_cloud_telematics/carma.config.js (hunk 1 above)
Comment (adev4a) [reply xil_cloud_telematics/carma.config.js:48]: Looks like the definitions in these files relate to carma-web-ui requests which is not being used with this config

File: xil_cloud_telematics/carma_docker.launch (hunk 3 above)
Comment (adev4a) [reply xil_cloud_telematics/carma_docker.launch:59]: These parameters were updated in this commit https://github.com/usdot-fhwa-stol/carma-config/commit/f3c6dc1cbf3cb090be6f35751fc54467aeb6cee4 to use_ros1_rosbag and use_ros2_rosbag. Looks like they were just moved to VehicleConfigParams.yaml

**Pull request merged**